### DISCUSSION TITLE
Public Constants Needed to be Public?

### DISCUSSION THREAD

**Discussion by don922** [cristianbuse/VBA-UserForm-MouseScroll#48] [General]
Hi Cristian,

I noticed that you have a couple of public constants defined in mousescroll.bas. I cannot find a reference to them outside of mousescroll.bas. Do they need to be public since they are only referenced in mousescroll.bas?

Don

**Comment by cristianbuse** [comment]
Hi Don,

No, they do not need to be public. They were used by the class at some point but that's not needed anymore.
I just made the change. Thank you for the suggestion!
